task  :  decnet-completion
network_model  :  enet2021
mode  :  train
dataset  :  nn
workers  :  4
epochs  :  20
criterion  :  l2
batch_size  :  1
learning_rate  :  0.001
weight_decay  :  1e-06
print_freq  :  10
resume  :
data_folder  :  /datasets
convolutional_layer_encoding  :  xyz
dilation_rate  :  2
val_h  :  352
val_w  :  608
min_depth_eval  :  0.1
max_depth_eval  :  80.0
kitti_crop  :  eigen_crop
train_datalist  :  train_dim_kitti.list
val_datalist  :  val_dim_kitti.list
root_folder  :  data/kitti_dataset/val_selection_cropped/
wandblogger  :  False
project  :  depth
entity  :  wandbdimar
STEP 2. Loading datasets...
Loaded 600 training files
Loaded 400 val files
  0%|                                                                         | 0/75 [00:00<?, ?it/s]
STEP 3. Loading model and metrics...
Loaded model enet2021 for decnet-completion
STEP 4. Training or eval stage...
STEP. Training block...
???
(0.0, 255.0, 101.54682922363281, 65.0)
torch.Size([8, 352, 608, 3])
Traceback (most recent call last):
  File "main.py", line 313, in <module>
    training_block()
  File "main.py", line 264, in training_block
    data_step = to_tensor(packed_data)
  File "/home/dim/depth_2022/features/custom_transforms.py", line 97, in __call__
    image, depth = transformation(image), transformation(depth)
  File "/home/dim/anaconda3/envs/pytorchenv/lib/python3.7/site-packages/torchvision/transforms/transforms.py", line 98, in __call__
    return F.to_tensor(pic)
  File "/home/dim/anaconda3/envs/pytorchenv/lib/python3.7/site-packages/torchvision/transforms/functional.py", line 117, in to_tensor
    raise ValueError('pic should be 2/3 dimensional. Got {} dimensions.'.format(pic.ndim))
ValueError: pic should be 2/3 dimensional. Got 4 dimensions.